gcsweb/cmd/gcsweb/gcsweb.go

```diff
@@ -111,24 +112,24 @@ func gatherOptions() options {
 
 func (o *options) validate() error {
 	if _, err := os.Stat(o.flIcons); os.IsNotExist(err) {
-		return fmt.Errorf("icons path '%s' doesn't exists.", o.flIcons)
+		return fmt.Errorf("icons path '%s' doesn't exists", o.flIcons)
```

Comment:
we should fix the grammar here while we're at it, `s/exists/exist/g` ?

Reply:
done

Reply:
might also make more sense to use `%q` instead of `'%s'` (unlikely, but if the path contains `'` this will be a pretty useless substring, at least %q should be fairly widely valid and unambiguous)

Reply:
done